integration-tests/test/native-eth.spec.ts

```diff
@@ -385,10 +377,9 @@ describe.skip('Native ETH Integration Tests', async () => {
       expect(
         await env.l2Wallet.provider.getBalance(env.l2Wallet.address)
       ).to.equal(initialBalance)
-      expect(receipt.events.length).to.equal(2)
```

Comment:
There is a big question here whether or not we want the fee to emit the erc20 events or not. It is ok to update the test in this way, but this is a decision we need to make. It is a difference from L1 but it is convenient to track tx fees over time

Reply:
My opinion is that it's best not to have an event here as it's quite hacky and it impacts developers who may be looking at transaction events by index.

Reply:
I'm not super opinionated but we need to be very clear when communicating because this will be a change in behavior for existing applications that made modifications for the existence of such an event